webapp/channels/src/i18n/en.json

```diff
@@ -4203,7 +4203,7 @@
   "multiselect.maxGroupMembers": "No more than 256 members can be added to a group at once.",
   "multiselect.numGroupsRemaining": "Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {group} other {groups}}. ",
   "multiselect.numMembers": "{memberOptions, number} of {totalCount, number} members",
-  "multiselect.numPeopleRemaining": "Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. ",
+  "multiselect.numPeopleRemaining": "You can't add more than 7 people. Please {createChannel} to include more people.",
```

Comment:
Looking at the code, you are not changing the default value of this string, but adding a new one. So this change should not be needed (the new string would have to be added here, though (to do that run `npm run i18n-extract`)).